### DISCUSSION TITLE
Postman Query Result and Admin UI Query Result are different

### DISCUSSION THREAD

**Discussion by Artur-Galstyan** [pocketbase/pocketbase#2557] [Q&A]
Hi everyone,

this is on the latest version.

On Postman I type in my query like so

`/api/collections/clip/records?filter=(clipId='01H151H2F4KZF7JYW6MZR4179Z')`

And the response is:

```json
{
    "page": 1,
    "perPage": 30,
    "totalItems": 0,
    "totalPages": 0,
    "items": []
}
```

But in the UI when I do the same, I find the record:
<img width="962" alt="Screenshot 2023-05-24 at 11 29 37" src="https://github.com/pocketbase/pocketbase/assets/63471891/977d3b4d-2930-43a6-9457-cb1ac4397a8a">

This shouldn't be the case, right? Is my query string wrong?

I also did this:

`/api/collections/clip/records`

which returned around 21K results (whereas the real DB has over 50K total results)

And after grabbing one of the results and running the filter query again, it found it.

I think the main issue is that 

`/api/collections/clip/records`

returns around 21K results but it should return around 50K (which is the total number of rows in that table)

**Comment by ganigeorgiev** [comment]
Are you sending the request authorized as admin in Postman?

**Reply by ganigeorgiev** [reply]
also make sure that the filter value is properly url encoded (_I'm not sure if Postman does this behind the scenes_):

```
/api/collections/clip/records?filter=clipId%3D%2701H151H2F4KZF7JYW6MZR4179Z%27
```

**Reply by Artur-Galstyan** [reply]
Hi, yes, I used Postmans Basic AUTH to login as admin. Since I am able to query "some" of all of the results, I don't expect that that could be the issue, right? 
Also, since I was able to get the other query result using a different clipId in the query, so it's probably not because of the encoding.

**Comment by Artur-Galstyan** [comment]
Sorry, my bad. I should have used the bearer token :(
